MEMBER_TRAVEL_FUND.md

```diff
@@ -18,6 +18,8 @@ and the Node.js Foundation projects.
 
 Requests must be made prior to the start of travel.
 Requests submitted after travel has started will not be approved.
+Candidates can only ask for reimbursement of prior approved items. In case of
```

Comment:
If this is a required step, then it should be logged be in the file, not in the description of the PR.
Because on Github, the description can be updated without approval and the previous version of the description can also be deleted.

So, we can not track it.

I'm not saying that someone would do it, but it can happen.

Also, I think we should also a template for PR description formate in this file. Something like this:

  | Cost
-- | --
Air Fare | $000
Lodging | $000
Ground Transportation | $000
Meals | $000
Entertain | $000
Phone | $000
Visa | $000
Other | $000
**Total** | **$000**







So that, people would know what they are/can requesting for and what they can ask for reimbursement in the future.

Reply:
I don't think this is moving any requirement around **what** you're able to request (or **where** to document it). It is putting some restriction on whether anyone can _retroactively_ add to an expense report though, which makes a lot of sense. 

> In case of new items added, the candidates must resubmit the updated request for a new approval.

I read this restriction as 

> "submit for reimbursement up to what you initially requested, and then go through the normal review process if you need more beyond that".

If we're planning ahead on the budget and have earmarked reimbursements for many folks, it's important that the funds paid out match closely (when possible) what was forecasted. So to me, all this clarifies is that we should do our diligence to plan ahead when making requests. Luckily, it also states changes after the fact can still happen as long as we have a second review.

Reply:
@WaleedAshraf yes this is correct - if you did not request the funds for X and you submitted a reimbursement for X, it needs to be approved in the PR and the PR needs to be updated.